test/Signature.ts

```diff
@@ -317,98 +224,86 @@ describe('Signature', function () {
         );
       });
       [messageWindow] = await waitForNewWindows(1);
-      await this.driver.switchTo().window(messageWindow);
-      await this.driver.navigate().refresh();
+      await browser.switchToWindow(messageWindow);
+      await browser.refresh();
     }
 
-    async function getPermissionRequestResult(this: mocha.Context) {
+    async function getPermissionRequestResult() {
       return JSON.parse(
-        await this.driver.executeScript<string>(() => {
+        (await browser.execute(() => {
```

Comment:
I have an error if i use generic parameter here.
I have to cast result inside script to avoid this error. (return result as string;)
So is it matters where i use cast?

Reply:
You can change `result` type from `unknown` to `any` then. It's in `test/utils/hooks.ts` file. Without extra parens it's usually much more readable, that's the reason :-)

Reply:
done